src/edgepi/eeprom/eeprom_constants.py

```diff
@@ -4,14 +4,19 @@
 
 # page write cycle time requirements https://www.onsemi.com/pdf/datasheet/cat24c256-d.pdf
 PAGE_WRITE_CYCLE_TIME=0.01
-DEFUALT_EEPROM_BIN = b"CmAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/\
-                       FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAASkAEKCg0AAIA/FQAAAAAK\
-                       Cg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAA\
-                       AAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/FQAAAAAKCg0AAIA/\
-                       FQAAAAAKCg0AAIA/FQAAAAAaEwoKDQAAgD8VAAAAABIFDQAAgD8iEwoKDQAAgD8VAAAAABIFDQAA\
-                       gD8qJAoOQ29uZmlnLVByaXZhdGUSEkNvbmZpZy1DZXJ0aWZpY2F0ZTIgCgxEYXRhLXByaXZhdGUS\
-                       EERhdGEtY2VydGlmaWNhdGU6DDIwMjIxMTEwLTAyMUIPRWRnZVBpLUJlYXJib25lSgxTTy0yMDIy\
-                       LTEwMjM="
+DEFUALT_EEPROM_BIN = b"CmAKCg0AAIA/FQAAAAASCg0AAIA/FQAAAAAaCg0AAIA/FQAAAAAiCg0AAIA/FQAAAAAqCg0AAIA/\
```

Comment:
I would just rename this to `DEFUALT_EEPROM_BIN_B64` to indicate the the content format is base-64.

Reply:
[f12061f](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/357/commits/f12061fdd6ca21808c3d7e722733295a8d7de261)